requirements.txt

```diff
@@ -275,14 +292,18 @@ sniffio==1.3.1
 sqlalchemy==2.0.30
     # via databases
 starlette==0.37.2
-    # via fastapi
-strawberry-graphql[fastapi]==0.229.0
+    # via
+    #   fastapi
+    #   strawberry-graphql
+strawberry-graphql[debug-server]==0.235.0
```

Comment:
Should we have both fastapi and debug-server as options, or does one supercede the other?